src/pages/workspaces/workspace/Dashboard/RightBoxSection.ts

```diff
@@ -3,29 +3,30 @@ import { div, h, h3 } from 'react-hyperscript-helpers';
 import Collapse from 'src/components/Collapse';
 import colors from 'src/libs/colors';
 import * as Style from 'src/libs/style';
+import { useLocalPref } from 'src/libs/useLocalPref';
 
 interface RightBoxSectionProps {
   title: string;
+  persistenceId: string; // persists whether or not the panel is open in local storage
   info?: ReactNode;
-  initialOpenState: boolean;
   afterTitle?: ReactNode;
-  onClick: () => void;
   children?: ReactNode;
 }
 
 export const RightBoxSection = (props: RightBoxSectionProps): ReactNode => {
-  const { title, info, initialOpenState, afterTitle, onClick, children } = props;
+  const { title, info, persistenceId, afterTitle, children } = props;
+  const [panelOpen, setPanelOpen] = useLocalPref<boolean>(persistenceId, false);
```

Comment:
`initialOpenState` was true for "Workspace information" (before persisted).

Reply:
I added an override for the default behavior, and set it to open for that panel.